gong/gong_to_hf_csv.py

```diff
@@ -0,0 +1,162 @@
+"""
+python gong_to_hf_csv.py
+
+Converts Gong transcription JSON to HF CSV
+"""
+# *********************************************************************************************************************
+
+# standard imports
+import json
+from datetime import datetime, timedelta
+import os
+
+# third Party imports
+import pandas
+import click
+
+
+@click.command()
+@click.option('-f', '--folder_path', type=str, required=True, help='Speechmatics json folder')
+@click.option('-e', '--expert_speaker_ids', type=str, required=True,
+              help='Provide the client speaker ID')
```

Comment:
help='Provide the client speaker ID' should be maybe "Provide the agents speaker IDs as comma delimited string"